.github/workflows/codeQLworkflow.yml

```
@@ -76,7 +76,7 @@ jobs:
       with:
         maven-version: ${{ inputs.mavenVersion }}
     - name: Build with Maven
-      uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3 # v1.0.1
+      uses: coactions/setup-xvfb@90473c3ebc69533a1a6e0505c36511b69c8c3135 # v1.0.1
```

according to https://github.com/coactions/setup-xvfb/issues/29#issuecomment-2100951480 we are supposed to use head - which is not v1.0.1:
```suggestion
      uses: coactions/setup-xvfb@6b00cf1889f4e1d5a48635647013c0508128ee1a
```

---

Done in 40c593b19bdd0c37ba3095f3b4fcdd3299c1dbf0, thank you!

---

this still has the v1.0.1 annotation

---

Forget it, I saw my mistake

---

Done in 45a2052f7d7ba212e0e179438a72dfcadf67eb38, thank you!